src/styles/card.css

```
@@ -0,0 +1,126 @@
+.card {
+  display: flex;
+  flex-flow: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+  background: url(../images/imac.jpeg);
+  background-size: cover;
+}
+
+.card__product_name {
+  margin-top: 40px;
+  margin-bottom: 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__product_code {
+  margin-top: 4px;
+  margin-bottom: 0;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+/* #region Reviews */
+.card__reviews_group {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+/* #region Stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background: url(../images/star.svg) no-repeat center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--5 .stars__star:nth-child(-n + 5),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--1 .stars__star:nth-child(-n + 1) {
```

you need only this one
```suggestion
.stars--4 .stars__star:nth-child(-n + 4) {
```